frappe/core/doctype/submission_queue/submission_queue.json

```diff
@@ -88,24 +88,14 @@
  ],
  "index_web_pages_for_search": 1,
  "links": [],
- "modified": "2023-01-23 12:45:53.997708",
+ "modified": "2023-08-28 18:48:56.242178",
  "modified_by": "Administrator",
  "module": "Core",
  "name": "Submission Queue",
  "naming_rule": "Random",
  "owner": "Administrator",
  "permissions": [
   {
-   "email": 1,
-   "export": 1,
-   "print": 1,
-   "read": 1,
-   "report": 1,
-   "role": "System Manager",
-   "share": 1
-  },
-  {
-   "if_owner": 1,
    "read": 1,
    "role": "All"
```

Comment:
handled this here https://github.com/frappe/frappe/pull/22224

Reply:
I'm not sure if that pr will fix this.

What i wanted to do: since we show the last failed attempt as a banner when submission has failed and since we have if owner checked with all perm, the user wont be able to see the last traceback by going to the subqueue doc if it was queued before by someone else.

I wanted to allow users to see all the subqueue docs where they have access to the reference doc.